src/Generator/DiffGenerator.php

```diff
@@ -146,6 +146,10 @@ private function createToSchema(): Schema
      */
     private function resolveTableName(string $name): string
     {
+        if ($this->platform->supportsSchemas()) {
+            return $name;
+        }
+
```

Comment:
After your change, I don't think the phpdoc comment is accurate anymore. Also, if the platform does not support schémas, surely the dot in a name should be interpreted literally? So... this method is no longer needed maybe?